metrics/web/test/common/presentation/widgets/metrics_fps_monitor_test.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/common/presentation/widgets/metrics_fps_monitor.dart';
+import 'package:statsfl/statsfl.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group("MetricsFPSMonitor", () {
```

Comment:
Should we test that the monitor got enabled once the `option` + `f` shortcut pressed?